packages/devextreme/js/__internal/ui/m_validation_group.ts

```diff
@@ -19,7 +19,7 @@ class ValidationGroup extends DOMComponent {
   _init() {
     // @ts-expect-error
     super._init();
-    ValidationEngine.addGroup(this);
+    ValidationEngine.addGroup(this, false);
```

Comment:
[missing test]
I changed it to true and no test is red. 
It should be tested in `validationGroup.tests`

Reply:
[c874986](https://github.com/DevExpress/DevExtreme/pull/27435/commits/c8749860b9d3db20aa0fb5abd1acd1063392b8c1)

Reply:
@jdvictoria  this note is still actual. If we change `false` to `true` here no test is red => your new test is evergreen